sonic-net/sonic-buildimage
# [Mellanox][sonic-bfb-installer] Adding Debug print messages

#### Why I did it
Addition of debug print messages to keep the SSH connection alive and to add per-dpu configuration capability with separate config files 

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

#### How to verify it
`sonic-bfb-installer.sh -r all -c a.conf,b.conf,c.conf,d.conf` sets the configuration individually for each dpus
The rshim devices are sorted before we apply the configuration
a.conf for rshim0, b.conf for the second and so on
We can also see debug pring messages during installation to keep the ssh connection alive



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
